python/cugraph/tests/utils.py

```diff
@@ -204,7 +211,7 @@ def create_obj_from_csv(
         )
 
     else:
-        raise TypeError(f"unsupported type: {obj_type}")
+        raise TypeError("unsupported type: {obj_type}")
```

Comment:
doesn't this still need to be an `f` string to properly expand `{obj_type}`?

Reply:
I think you're right. I removed it because it was generating a flake8 error (locally)